### ISSUE TITLE
Inconsistency in Nuclear Segmentation - DeepWatershed.ipynb notebook

### ISSUE THREAD

**Issue by AlexInJar** [vanvalenlab/deepcell-tf#576]
**Describe the bug**
I have been trying to reproduce results in Nuclear Segmentation - DeepWatershed.ipynb in training notebook. However, I think there are some problems in the metric calculations:

<img width="454" alt="Screen Shot 2022-01-14 at 10 09 38 AM" src="https://user-images.githubusercontent.com/60033670/149439371-4baa763e-cf5d-4729-9989-225135fd2399.png">

As shown here, should not the recall be 5792/6244 = 92.7601%

Also, I think there are something inconsistent in the last cell, should it be not that the 
outputs = prediction_model.predict(X_test) in the follow line:
<img width="624" alt="Screen Shot 2022-01-14 at 10 13 17 AM" src="https://user-images.githubusercontent.com/60033670/149439809-507f89ec-69ad-4550-88fe-523b36728253.png">

**Environment:**
 - OS: Ubuntu 16.04
 - Python Version: 3.8.5
 - Tensorflow: 2.4.1
 - deepcell: 0.11.0

**Comment by willgraf** [comment]
I believe that the recall issue has been fixed in vanvalenlab/deepcell-toolbox#118.

---

Yes, we should probably change it to `prediction_model.predict` rather than `model.predict`. However, both models have the same weights so the outputs should be the same.

The prediction model drops an unnecessary semantic head. The foreground/background head can be useful in training, but the outputs are not used in post-processing.

**Comment by AlexInJar** [comment]
Thank you for your comment.
May I further ask if you have any references for how the Mesmer was originally trained? 
I have been trying to retrain mesmer using data from https://datasets.deepcell.org/, and the code in the notebook panopticnet. I used only one channel for the nuclear channel and trained for 30 epoch, however, the result is different, with a precision and recall of 30%.

May I ask if you may provide code where you originally trained mesmer?

**Comment by ngreenwald** [comment]
All of the details from the paper are in the [publications repo](https://github.com/vanvalenlab/publication-figures). Here's the notebook for training the model:
https://github.com/vanvalenlab/publication-figures/blob/master/2021-Greenwald_Miller_et_al-Mesmer/Mesmer_training_notebook.ipynb

**Comment by AlexInJar** [comment]
Thank you very much! That is what I was looking for.
